.github/workflows/test.js.yml

```
@@ -3,6 +3,7 @@ name: Run yarn test
 on:
   push:
     branches-ignore:
+      - '*'
```

do you want to disable tests for now ?

---

yes, pr desc
>  Tests are disabled until fixed in future PR/s

---

even though, I would let the build fail, and manually merge if necessary for now, but better to know that tests don't pass. what do u think ?

---

Sure, for me sounds ok
I will let you handle the mering and revert this

👍

---

you can merge, you are admin as well